### DISCUSSION TITLE
Use two line breaks for steps

### DISCUSSION THREAD

**Discussion by dubadub** [cooklang/spec#65] [Ideas]
_Reported via email_

Don’t parse single line breaks as separate steps for the Steps display in the app. Single line breaks are generally used in plain text formats like email and Markdown to keep paragraphs from displaying as one long line on “dumb” clients like cat. Two line breaks generally indicate a paragraph break. Wrapping paragraphs with single line breaks at somewhere around 80 characters also makes git diffs somewhat nicer to work with

**Comment by DanMacKay1961** [comment]
Agreed. This is following the "Make it easy to write text files" principle.  Requiring long steps to have super  long lines doesn't really fit this.

**Comment by dncnmcdougall** [comment]
I agree with this. I was thinking that this could be a section in the spec. As we have a `#cookware` section we could have a `#steps` section.
I'll create a pull request for this.

**Comment by lashomb** [comment]
I had tried this hoping it would work even before seeing this... seems like an intuitive addition.

**Comment by dubadub** [comment]
This is now part of the main specification, closing the discussion.

**Comment by dncnmcdougall** [comment]
Excellent, thank you.
